lib/internal/dns/utils.js

```diff
@@ -61,7 +61,7 @@ class Resolver {
   }
 
   getServers() {
-    return ArrayPrototypeMap(this._handle.getServers(), (val) => {
+    return ArrayPrototypeMap(this._handle.getServers() || [], (val) => {
```

Comment:
In which case `getServers()` will return `null` or `undefined`?

Reply:
I think when `server_array->Set(env->context(), i, Array::New(env->isolate(), ret, arraysize(ret))) .IsNothing()` is true in `GetServers` of `cares_wrap.cc`.

Reply:
I can't reproduce it using javascript.

Reply:
Yes, i think this is difficult to trigger.